CHANGELOG.md

```diff
@@ -18,6 +18,8 @@
   `utf_codepoint_to_int` functions.
 - Fixed `string.inspect`'s escaping of `"`, `\`, `\n`, `\r`, `\r\n`, and `\t`,
   which in turn fixes `io.debug`'s output of such strings.
+- The `list` module gains the `group` function.
+- The `option` and the `map` module gain independence from the `list` module
```

Comment:
This is an invisible internal change so shouldn't go in the changelog

Reply:
Removed the internal changes in the changelog file